src/frontend/src/routes/(new-styling)/recovery-phrase/+page.svelte

```diff
@@ -53,6 +54,9 @@
   >(undefined);
 
   let continueInProgress = $state(false);
+  let recoveryError = $state(false);
```

Comment:
Nit: This variable could probably use a comment to indicate what `true` means e.g.
```suggestion
  let recoveryError = $state(false); // Recovery phrase fails checksum validation
```